ctru-rs/src/console.rs

```diff
@@ -66,6 +79,12 @@ pub struct Console<'screen> {
     screen: RefMut<'screen, dyn ConsoleScreen>,
 }
 
+/// Send output from stderr to the specified [`Destination`]. This function can be used to capture error and panic messages with `GDB` or other debuggers.
+#[doc(alias = "consoleDebugInit")]
+pub fn redirect_stderr(dest: Destination) {
+    unsafe { consoleDebugInit(dest as _) }
+}
```

Comment:
This looks fine to me offhand, I wonder how it behaves if you also use `redirect_to_3dslink` with `stderr = true` ? I think they use the same mechanism under the hood, so maybe whichever you call first gets overridden by the second one?

Reply:
I took a look inside libctru and the mechanisms are dfferent. `consoleDebugInit` [changes the underlying devoptab for stderr](https://github.com/devkitPro/libctru/blob/master/libctru/source/console.c#L758-L761) and `3dslinkConnectToHost` [replaces stderr's file descriptor via `dup2`](https://github.com/devkitPro/libctru/blob/master/libctru/source/3dslink.c#L41-L45). So 3dslink redirection will always take priority over console redirection.

Reply:
That behaviour should be properly documented.